src/Nethermind/Nethermind.Optimism/OptimismCostHelper.cs

```diff
@@ -141,6 +146,35 @@ public UInt256 ComputeOperatorCost(long gas, BlockHeader header, IWorldState wor
         }
     }
 
+    public UInt256 ComputeDAFootprint(Block block, IWorldState worldState)
+    {
+        // https://specs.optimism.io/protocol/jovian/exec-engine.html#scalar-loading
+        const int scalarPosition = 12;
+        ReadOnlySpan<byte> span = worldState.Get(_daFootprintGasScalarSlot);
+
+        if (span.IsEmpty)
+            return 0;
+
+        var daFootprintGasScalar = ReadUInt16BigEndian(span[scalarPosition..]);
+
+        UInt256 footprint = UInt256.Zero;
+
+        foreach (Transaction tx in block.Transactions)
+        {
+            if (tx.Type == TxType.DepositTx)
+                return 0;
```

Comment:
We don't abort with 0 but instead skip Deposit transactions.

```suggestion
                continue;
```